fbfmaproom/fbfmaproom.py

```diff
@@ -106,14 +105,60 @@ def units(self):
             self._units = self.open().attrs.get('units')
         return self._units
 
+    def open(self, val_min=None, val_max=None):
+        path = data_path(self.path)
+        try:
+            ds = xr.open_zarr(path, consolidated=False)
+        except Exception as e:
+            raise Exception(f"Couldn't open {path}") from e
+        ds = ds.rename({
+            v: k
+            for k, v in self.var_names.items()
+            if v is not None and v != k
+        })
```

Comment:
This is just a comment but every day Python's syntactic special cases never cease to amaze me